mathquill4quill.css

```
@@ -52,9 +52,25 @@
   overflow: auto;
 }
 
+.mathquill4quill-history-container-with-delete-button {
+  width: 318px;
+}
+
 .mathquill4quill-latex-input {
   visibility: hidden !important;
   padding: 0 !important;
   border: 0 !important;
   width: 0 !important;
 }
+
+.mathquill4quill-history-delete-button:after {
```

Nit: this should use the double colon to make stylelint happy
```suggestion
.mathquill4quill-history-delete-button::after {
```